### PR TITLE
Fix small mistakes in Adaround example script comments

### PR DESCRIPTION
**Pull request by mstaczek**
Issue: #2208

### CODE REVIEW

File: Examples/torch/quantization/adaround.py [hunk 1]
```diff
@@ -120,7 +120,7 @@ def evaluate(self, model: torch.nn.Module, iterations: int = None, use_cuda: boo
 
 def apply_adaround_and_find_quantized_accuracy(model: torch.nn.Module, evaluator: aimet_common.defs.EvalFunction,
                                                data_loader: torch_data.DataLoader, use_cuda: bool = False,
-                                               logdir: str = '') -> Tuple[torch.nn.Module, float]:
+                                               logdir: str = '') -> float:
```
Comment (quic-mangal) [inline Examples/torch/quantization/adaround.py:122]: Please remove line 49 `from typing import Tuple` since this import is not being used anymore

**Comment by quic-mangal** [comment]
@mstaczek Thanks for catching it and making the changes.

File: Examples/torch/quantization/adaround.py (hunk 1 above)
Comment (mstaczek) [reply Examples/torch/quantization/adaround.py:122]: Right, didn't notice it. Removed.

**Review by quic-mangal — APPROVED** [review]

**Comment by quic-akhobare** [comment]
Appreciate your contribution @mstaczek. Thanks.

**Pull request merged**